deploy/crds/interconnectedcloud_v1alpha1_qdrouterd_crd.yaml

```diff
@@ -136,6 +169,28 @@ spec:
                   - port
                 type: object
               type: array
+            interEdgeConnectors:
+              items:
+                properties:
+                  cost:
+                    format: int32
+                    type: integer
+                  host:
+                    type: string
+                  name:
+                    type: string
+                  port:
+                    format: int32
+                    type: integer
+                  routeContainer:
```

Comment:
Is the routeContainer relevant here? Isn't the underlying role implied by the fact that this is an edge connector?

Reply:
Needed for crd validation, will take action to look into contextualizing the validation...

Reply:
Of course, I forgot it is in the shared Listener definition. Leave it for now. (Ultimately I think we need to reconsider how we represent route-container connectors/listeners)